iroh-net/src/discovery/pkarr.rs

```diff
@@ -174,6 +181,48 @@ impl PublisherService {
     }
 }
 
+/// Resolve node info using a pkarr relay.
+#[derive(derive_more::Debug, Clone)]
+pub struct PkarrResolver {
```

Comment:
So this is for when you want to resolve directly via the relay? Yeah, makes sense to have this.